Aquality.Selenium/src/Aquality.Selenium/Aquality.Selenium.csproj

```diff
@@ -19,6 +19,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="Resources\JavaScripts\OpenInNewTab.js" />
+    <None Remove="Resources\JavaScripts\OpenNewTab.js" />
```

Comment:
not sure, why do we need these "None Remove" items?

Reply:
fixed